### PR TITLE
Comments + no "successful", ZIP, deletion

### PR DESCRIPTION
**Pull request by comradekingu**
Assumptions were made.

### CODE REVIEW

**Comment by dachary** [comment]
The failures are real: you changed strings that are also used in tests and they also need to adapt.

File: securedrop/journalist.py [hunk 1]
```diff
@@ -601,8 +601,8 @@ def reply():
     generic error message is flashed on the client.
 
     Returns:
-       flask.Response: The user is redirected to the same Source
-           collection view, regardless if the Reply is created
+       flask.Response: The user is redirected to the same source
+           collection view, regardless if the reply is created
```
Comment (redshiftzero) [inline securedrop/journalist.py]: These capitalizations in the original comments were intentional - they refer to the `Source` and `Reply` objects

**Comment by dachary** [comment]
will push another PR to squash the changes together

**Comment by dachary** [comment]
https://github.com/freedomofpress/securedrop/pull/2265 continues this PR

**Pull request closed without merging**